dftimewolf/lib/collectors/grr_hosts.py

```diff
@@ -103,6 +103,39 @@ def _GetClientBySelector(self, selector):
       self.ModuleError('Could not get client for {0:s}'.format(
           selector), critical=True)
 
+    def _SeenLastMonth(timestamp):
+      """ Take a UTC timestamp and check if it is in the last month.
+
+      Args:
+        timestamp (int): A timestamp in microseconds.
+
+      Returns:
+        boolean: True if the timestamp is in last month from now.
+      """
+      last_seen_datetime = datetime.datetime.utcfromtimestamp(
+          timestamp / 1000000)
+      diff_year_in_month = (datetime.datetime.utcnow().year
+          - last_seen_datetime.year) * 12
+      diff_month_in_month = (datetime.datetime.utcnow().month
+          - last_seen_datetime.month)
+      diff_month = diff_year_in_month + diff_month_in_month
+      return bool(diff_month < 1)
```

Comment:
A simpler way to do this is to do a check like:

```
month_ago = datetime.datetime.utcnow() - datetime.timedelta(30)  # 30 days before now()
return  last_seen_datetime > month_ago
```

Reply:
Done